dev/Bootstrap/CS/Microsoft.WindowsAppRuntime.Bootstrap.Net/Bootstrap.cs

```diff
@@ -1,4 +1,13 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// ------------------------------------------------------------------------------
+// <auto-generated>
```

Comment:
This is NOT a generated file. It's artisanal hand-crafted code

Reply:
It is said a robot wrote this code. <g>

Reply:
> It is said a robot wrote this code. `<g>`

I've been called many things (some of them even printable) but first time a robot :P